python/lsst/ip/isr/photodiode.py

```diff
@@ -297,10 +297,17 @@ def integrateChargeSum(self):
         # The current per interval to use for baseline subtraction
         # without assuming all of the dt values are the same:
         current = charge/dt
-        # For the baseline current level, select current values < 5%
-        # of the maximum, measured relative to the overall minimum.
+        # To determine the baseline current level, exclude points with
+        # signal levels > 5% of the maximum (measured relative to the
+        # overall minimum), and extend that selection 2 entries on
+        # either side to avoid otherwise low-valued points that sample
+        # the signal ramp and which should not be included in the
+        # baseline estimate.
         dy = np.max(current) - np.min(current)
-        index = np.where(current < dy/20. + np.min(current))
-        bg_current = np.sum(charge[index])/np.sum(dt[index])
+        signal = np.where(current > dy/20. + np.min(current))
+        imin = signal[0][0] - 2
```

Comment:
What happens if the first point is at the edge?  Do we need to make sure this doesn't go below 0?  (Similar for the next line at the max).

Reply:
Both `list(range(0, imin))` and `list(range( imax, len(current)))` return empty lists when `imin < 0` and `imax > len(current)`, respectively, which is what we want, I think.  Same appears to be true for `np.arange`.